ambry-router/src/main/java/com/github/ambry/router/PutOperation.java

```diff
@@ -1613,6 +1649,13 @@ int fillFrom(ByteBuf channelReadBuf) {
     void checkAndMaybeComplete() {
       boolean done = false;
       // Now, check if this chunk is done.
+      if (chunkException != null && chunkException.getErrorCode() == RouterErrorCode.BlobCorrupted) {
+        logger.error("{} : Put chunk {} failed due to corruption. Failing entire operation", loggingContext,
```

Comment:
can we add a new metric for this error?

Reply:
I have a metric `putBlobCRCMismatchCount` which gets incremented when `onPutBlobError` is called during completion of the operation. Will that be good for us?